evol/population.py

```diff
@@ -78,6 +78,20 @@ def generate(cls, init_func, eval_func, size=100) -> 'Population':
         chromosomes = [init_func() for _ in range(size)]
         return cls(chromosomes=chromosomes, eval_function=eval_func)
 
+    @property
+    def individual_weights(self):
+        try:
+            min_fn = min(individual.fitness for individual in self)
+            max_fn = max(individual.fitness for individual in self)
+        except TypeError:
+            raise RuntimeError('Individual weights can not be computed if the individuals are not evaluated.')
+        if min_fn == max_fn:
+            return [1] * len(self)
+        elif self.maximize:
+            return [(individual.fitness - min_fn)/(max_fn-min_fn) for individual in self]
```

Comment:
Note: we may decide to not normalise the weights to [0, 1], as that isn't required by `random.choices`. On the other hand, this is easier to test and to re-use.